inc/stream/namespace.php

```diff
@@ -8,6 +8,7 @@
 namespace Altis\Security\Stream;
 
 use Altis;
+use function HM\Platform\XRay\get_root_trace_id;
```

Comment:
We're moving away `use function` statements in favour of using the namespace so it's clearer where a function is from at the point of use, so in this case it'd be `use HM\Platform\XRay` and `XRay\get_root_trace_id()`